code/__DEFINES/dcs/signals.dm

```diff
@@ -45,6 +45,10 @@
 #define COMSIG_WEATHER_START(event_type) "!weather_start [event_type]"
 #define COMSIG_WEATHER_WINDDOWN(event_type) "!weather_winddown [event_type]"
 #define COMSIG_WEATHER_END(event_type) "!weather_end [event_type]"
+/// An alert of some form was sent (datum/alert_handler/source, alert_type, area/source_area))
```

Comment:
```suggestion
/// An alert of some form was sent (datum/alert_handler/source, alert_type, area/source_area)
```

Can we also be clear what "alert" means here? From the PR title and from this documentation, I had no idea you were talking about like...air alarms?

Reply:
That comment goes for all your doc comments here.

Reply:
I use alert and alarm interchangeably in this pr, it's really more like alarms honestly. I've renamed my code, though honestly now that I've thought about it I'm not sure how clarifying that is.

In general the system is for stuff that wants to alert a lot of other objects in the world when something about it changes, alongside passing along a list of cameras. This is primarily used by alert consoles and silicons. It has some distinguishing features outside of just having a global list of "alarms", primarily supporting new alarm listeners not knowing about old alarms.

Reply:
I'll elaborate in the datum autodoc